plugins/by-name/jupytext/deprecations.nix

```diff
@@ -0,0 +1,13 @@
+lib: {
+  imports = [
+    # TODO: introduced 2025-10-25: remove after 26.05
+    (lib.mkRenamedOptionModule
+      [
+        "plugins"
+        "jupytext"
+        "python3Dependencies"
+      ]
+      [ "extraPython3Packages" ]
+    )
```

Comment:
Technically, the old option was gated behind the plugin's `enable` option before, while this alias will copy the option's definitions regardless of whether the plugin is enabled.